spec/mailers/previews/error_preview.rb

```diff
@@ -0,0 +1,3 @@
+# Preview all emails at http://localhost:3000/rails/mailers/error
+class ErrorPreview < ActionMailer::Preview
```

Comment:
Can this be removed?

Reply:
This was autogenerated when I created the new mailer. I don't think it does anything but I'll remove it anyway.